ccm_web/client/src/components/ValidationErrorTable.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useEffect, useState } from 'react'
+import { IconButton, makeStyles, Paper, Table, TableBody, TableCell, TableContainer, TableFooter, TableHead, TablePagination, TableRow, useTheme } from '@material-ui/core'
+import { KeyboardArrowRight, KeyboardArrowLeft, LastPage as LastPageIcon, FirstPage as FirstPageIcon } from '@material-ui/icons'
+import PropTypes from 'prop-types'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexShrink: 0,
+    marginLeft: theme.spacing(2.5)
+  }
+}))
+
+interface ValidationError {
+  rowNumber: number
+  message: string
+}
+
+interface ErrorTableProps {
+  invalidations: ValidationError[]
+}
+
+interface TableHeaderColumnInfoShouldUseMatUIType {
+  id: keyof ValidationError
+  label: string
+  minWidth: number
+  align?: 'left' | 'right' | undefined
+}
+
+const columns: TableHeaderColumnInfoShouldUseMatUIType[] = [
+  { id: 'rowNumber', label: 'Row Number', minWidth: 25 },
+  { id: 'message', label: 'Error', minWidth: 100 }
+]
+
+function ErrorTable (props: ErrorTableProps): JSX.Element {
+  const classes = useStyles()
+  const [tableRows, setTableRows] = useState<ValidationError[]>([])
+  const [page, setPage] = React.useState<number>(0)
+  const [rowsPerPage, setRowsPerPage] = React.useState(5)
+
+  useEffect(() => {
+    setTableRows(props.invalidations.map(i => {
+      return { rowNumber: i.rowNumber, message: i.message }
+    }).sort((a, b) => (a.rowNumber < b.rowNumber ? -1 : 1))
+    )
+  }, [props.invalidations])
```

Comment:
For the error when the Current and final grade don't match I feel, The UI currently shows rows number error is occurring, but those error row numbers don't exactly match the row number in the file. For Example if error is show as Row 1 in the CCM UI, in the File the row number is actually Row 3. I think header ( 2 rows) need to be added when match the row numbers.

![Screen Shot 2021-05-06 at 7 45 13 AM](https://user-images.githubusercontent.com/8579775/117294039-2b264380-ae40-11eb-9bd5-27510435822f.png)
![Screen Shot 2021-05-06 at 7 53 00 AM](https://user-images.githubusercontent.com/8579775/117294050-2e213400-ae40-11eb-80f0-885c5d90c78b.png)

Reply:
Yeah it's showing the  data row, I can offset by 2 to account for headers if that's what we want.

Reply:
Yes, It make sense to match the row for better feedback to user